x-pack/plugin/data-frame/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/dataframe/integration/DataFramePivotRestIT.java

```diff
@@ -224,6 +227,36 @@ public void testDateHistogramPivot() throws Exception {
         assertOnePivotValue(dataFrameIndex + "/_search?q=by_day:2017-01-15", 3.82);
     }
 
+    @SuppressWarnings("unchecked")
+    public void testPreviewTransform() throws Exception {
+        final Request createPreviewRequest = new Request("POST", DATAFRAME_ENDPOINT + "_preview");
+
+        String config = "{"
+            + " \"source\": \"reviews\","
+            + " \"id\": \"doesnot-matter\","
+            + " \"dest\": \"doesnot-matter\",";
+
+        config += " \"pivot\": {"
+            + "   \"group_by\": ["
```

Comment:
the `[`, `]` need to be removed

Reply:
@hendrikmuhs definitely! Was waiting on your PR to be merged :D